src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java

```diff
@@ -1,12 +1,12 @@
 /**
- * Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
  *
  * This file is part of fiware-cygnus (FI-WARE project).
  *
- * cosmos-injector is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+ * fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
```

Comment:
Yes, I noticed that :( This is like a plague, there is always some reminiscence from the past... Please, create the issue. Thanks!